app/models/concerns/katalyst/content/version.rb

```diff
@@ -13,6 +13,14 @@ module Version
         # rubocop:enable Rails/ReflectionClassName
 
         attribute :nodes, Katalyst::Content::Types::NodesType.new, default: -> { [] }
+
+        validate :ensure_items_exists
+      end
+
+      def ensure_items_exists
+        if nodes.detect { |node| parent.items.find_by(id: node.id).nil? }
+          errors.add(:items, "invalid")
```

Comment:
I think this should be using a symbol so it can be translated with i18n, instead of the string "invalid"
e.g. something like

`errors.add(:items, :invalid_katalyst_content_item)`